datasources/src/main/java/org/opensearch/sql/datasources/transport/TransportCreateDataSourceAction.java

```diff
@@ -56,9 +60,15 @@ protected void doExecute(
     try {
       DataSourceMetadata dataSourceMetadata = request.getDataSourceMetadata();
       dataSourceService.createDataSource(dataSourceMetadata);
+      String responseContent =
+              new JsonResponseFormatter<String>(PRETTY) {
+                @Override
+                protected Object buildJsonObject(String response) {
+                  return response;
+                }
+              }.format("Created DataSource with name " + dataSourceMetadata.getName());
```

Comment:
Is this working? I feel it gives the same response. What are we trying to fix here?

Reply:
For context:: https://stackoverflow.com/questions/42981632/return-simple-string-in-web-api-producing-application-json

If we want to send a json response: we should change the response to 

{
       "status": "Created DataSource with name {name}"
}

Reply:
Addressed in: [96f3ef0](https://github.com/opensearch-project/sql/pull/2156/commits/96f3ef0098a4fd9207c86ebe32bffb0950d55019)